downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -234,7 +237,13 @@ class Git : VersionControlSystem(), CommandLineTool {
             logger.warn { "Failed to fetch everything: ${it.collectMessages()}" }
         }.mapCatching {
             // TODO: Migrate this to JGit once https://bugs.eclipse.org/bugs/show_bug.cgi?id=383772 is implemented.
-            run("checkout", revision, workingDir = git.repository.workTree)
+            run("checkout", revision, workingDir = workingTree.workingDir)
```

Comment:
Why was the `workingDir` argument change necessary?

Reply:
It's wasn't necessary, I changed it for consistency as the `workingTree` is now available as a function parameter.

Reply:
Could you please briefly mention this in the commit message, to make clear that this part of the diff is not supposed to change behavior?

Reply:
I have moved it to a separate commit.

Reply:
Even better, thanks!